Library/Formula/imply.rb

```diff
@@ -0,0 +1,21 @@
+class Imply < Formula
+  desc "Exploratory Analytics Powered By Druid"
+  homepage "http://imply.io/"
+  url "http://static.imply.io/release/imply-#{version}.tar.gz"
+  version "1.1.0"
+  sha256 "d06add07832c4253f0f3722b1625d034c52f3d9041e59779a277b72a84d1ce53"
+
+  bottle :unneeded
+
+  def install
+    libexec.install Dir["*"]
+    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    ohai "Start up services with:"
+    ohai "    supervise -c #{libexec}/conf/supervise/quickstart.conf"
+    ohai "See http://imply.io/docs/latest/quickstart for more information"
+  end
+
+  test do
+    File.exist? "#{bin}/supervise"
```

Comment:
Apart from testing for an installed binary, I'm unable to think of a better way to write this test. More substantial tests require starting up the Imply services, which I am unable to do within the test's sandbox.